### PR TITLE
Drop remote_execution plugin-related permissions from assertions if the plugin not installed

### PR DESCRIPTION
**Pull request by rplevka**
Sat 6.2 comes with no `remote_execution` plugin enabled by default, resulting in some corresponding permissions missing.
This PR will check whether the plugin package is installed and if not, it will exclude the corresponding permissions from the test.

### CODE REVIEW

**Comment by rplevka** [comment]
```bash
+ pytest -k test_positive_search_by_resource_type tests/foreman/api/test_permission.py
++ which py.test
+ /home/jenkins/shiningpanda/jobs/ad0ef6b5/virtualenvs/d41d8cd9/bin/py.test -v --junit-xml=foreman-results.xml -m 'not stubbed' -k test_positive_search_by_resource_type tests/foreman/api/test_permission.py
============================= test session starts ==============================
platform linux2 -- Python 2.7.10, pytest-2.9.1, py-1.4.31, pluggy-0.3.1 -- /home/jenkins/shiningpanda/jobs/ad0ef6b5/virtualenvs/d41d8cd9/bin/python2.7
cachedir: .cache
rootdir: /home/jenkins/workspace/satellite6-standalone-automation, inifile: 
plugins: xdist-1.14
collecting ... collected 7 items
tests/foreman/api/test_permission.py::PermissionTestCase::test_positive_search_by_resource_type <- robottelo/decorators.py PASSED
generated xml file: /home/jenkins/workspace/satellite6-standalone-automation/foreman-results.xml 
6 tests deselected by "-ktest_positive_search_by_resource_type -m 'not stubbed'" 
==================== 1 passed, 6 deselected in 7.50 seconds ====================
+ exit 0
Deleting 1 temporary files
Archiving artifacts
Recording test results
Started calculate disk usage of build
Finished Calculation of disk usage of build in 0 seconds
Started calculate disk usage of workspace
Finished Calculation of disk usage of workspace in 0 seconds
Finished: SUCCESS
```

File: robottelo/constants.py [hunk 1]
```diff
@@ -326,22 +326,16 @@
         'configuration',
         'create_arf_reports',
         'create_containers',
-        'create_job_invocations',
-        'create_job_templates',
```
Comment (rplevka) [inline robottelo/constants.py]: I removed these (and some more) as they were duplicate (their duplicates can be found in the appropriate sections below in the list)

**Comment by elyezer** [comment]
ACK

File: tests/foreman/api/test_permission.py [hunk 2]
```diff
@@ -42,6 +42,14 @@ def setUpClass(cls):
             cls.permissions[None].remove('destroy_arf_reports')
             cls.permissions[None].remove('view_arf_reports')
             cls.permissions[None].remove('create_arf_reports')
+        result_re = ssh.command(
+            'rpm -qa | grep rubygem-foreman_remote_execution'
+        )
+        if result_re.return_code != 0:
```
Comment (elyezer) [inline tests/foreman/api/test_permission.py]: You can reuse result variable here

**Comment by abalakh** [comment]
ACK

File: tests/foreman/api/test_permission.py (hunk 2 above)
Comment (rplevka) [inline tests/foreman/api/test_permission.py]: @elyezer - done

**Pull request merged**